Reply/app/src/main/java/com/materialstudies/reply/ui/email/EmailFragment.kt

```diff
@@ -62,6 +65,23 @@ class EmailFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         binding = FragmentEmailBinding.inflate(inflater, container, false)
+
+        lifecycleScope.launch {
+            EmailUtils.selectedEmailState.collect { id ->
```

Comment:
Change to EmailStore.selectedEmailId

Reply:
Also in this file on L95 you'll have to update to use `EmailStore.selectedEmailId.value`